ci.yml

```diff
@@ -38,6 +38,8 @@ support:
    - typescript@4.5
    - typescript@4.6
    - typescript@4.7
+   - typescript@4.8
+   - typescript@4.9
```

Comment:
I vaguely remember that you had some CI generator project, right? Is this what this file is for?